app/Http/Controllers/ProductController.php

```diff
@@ -82,16 +89,24 @@ public function destroy(Product $product)
     {
         $product->delete();
 
-        return $this->sendSuccess([],'Product deleted successfully');
+        return $this->sendSuccess([], 'Product deleted successfully');
     }
 
     /**
      * @return Response
      */
     public function uploadSheet()
     {
-        Excel::import(new ProductsImport(),Request::file('file'));
+        Excel::import(new ProductsImport(), Request::file('file'));
+
+        return $this->sendSuccess([], 'Product Uploaded successfully');
+    }
 
-        return $this->sendSuccess([],'Product Uploaded successfully');
+    public function fetchByRank(NewOrderFilter $filter, ProductService $productService, HttpRequest $request)
```

Comment:
I think it is better to state what you are fetching by rank. e.g fetchLeastAndMostOrderedProduct (explains what the method does better)

Reply:
done